src/registration/tests/utils.py

```diff
@@ -3,13 +3,24 @@
 """
 """
 __author__ = 'Alisue <lambdalisue@hashnote.net>'
+from django.core.management.base import CommandError
+
 
 def recall_syncdb():
     """call ``syncdb`` command to create tables of new app's models"""
-    from django.db.models import loading
     from django.core.management import call_command
-    loading.cache.loaded = False
-    call_command('syncdb', interactive=False, verbosity=0, migrate=False, migrate_all=True)
+    try:
+        from django.db.models import loading
+        loading.cache.loaded = False
+    except ImportError:
+        # In Django1.9, django.db.models.loading is removed
+        pass
+    try:
+        call_command('syncdb', interactive=False, verbosity=0, migrate=False, migrate_all=True)
+    except CommandError:
+        # In Django1.9, syncdb command is removed
+        pass
```

Comment:
Are you sure that only Django 1.9 does not require `syncdb`?
It would be nice `syncdb` only called on Django which requires `syncdb`